src/Tizen.System/Device/PmQos.cs

```diff
@@ -0,0 +1,88 @@
+/*
+* Copyright (c) 2020 Samsung Electronics Co., Ltd All Rights Reserved
+*
+* Licensed under the Apache License, Version 2.0 (the License);
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an AS IS BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+using System;
+
+using Tizen.Common;
+
+namespace Tizen.System
+{
+    /// <summary>
+    /// The PmQos class provides the methods to control the PmQos.
+    /// </summary>
+    /// <remarks>
+    /// The PmQos API provides the way to control the PmQos. It supports to increase cpu clock within input timeout.
+    /// </remarks>
+    /// <privilege>
+    /// </privilege>
+    /// <since_tizen> 6 </since_tizen>
```

Comment:
since_tizen should be 8, it means API Level 8
```suggestion
    /// <since_tizen> 8 </since_tizen>
```

Reply:
Done